flask/boxwise_flask/graph_ql/query_defs.py

```diff
@@ -5,16 +5,21 @@
     """
     type Query {
         hello: String!
-        allBases: [Base]
-        orgBases(orgId: Int): [Base]
-        base(id: Int!): Base
+        bases: [Base!]!
+        base(id: ID!): Base
+        organisation(id: ID!): Organisation
+        organisations: [Organisation!]!
         allUsers: [User]
```

Comment:
why the inconsistency between `allUsers `vs. `user `and `bases `instead of `allBases`?

Reply:
The user-related queries are still to be updated. I suggest we name them `user` and `users` then

Reply:
As a naming convention for GraphQL queries I try to follow the style of most examples, and the official docs: singular noun for single resource, plural noun for list of resources, no additional verbs (like get, fetch, ...), no additional adverbs (as I said, the 'all' in `allUsers` will soon be dropped ;-)